CHANGELOG.md

```diff
@@ -2,11 +2,25 @@
 
 Certbot adheres to [Semantic Versioning](https://semver.org/).
 
-## 0.36.0 - master
+## 0.36.1 - master
 
 ### Added
 
-* Added missing error types from RFC8555 to acme 
+*
+
+### Changed
+
+* Updated acme messages.py to adhere to RFC8555
```

Comment:
For what it's worth, I personally like your previous wording better as I'm not sure messages.py is 100% compliant with RFC8555.

Reply:
Ok, I fixed the changelog. I used my old wording, I agree with what you said.

Reply:
Great! I haven't seen any updates to the code in this PR yet so after you push the changes, I'll take another look.

Reply:
I pushed into the wrong branch. Fixing that now.